metron-platform/metron-hbase/metron-hbase-common/src/main/java/org/apache/metron/hbase/ColumnList.java

```diff
@@ -76,6 +80,30 @@
     public long getTs() {
       return ts;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Adding the obvious equals/hashCode/toString to these objects made implementing the `FakeHBaseClient` much simpler.